.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: objective-c
-osx_image: xcode8
+osx_image: xcode8.1
```

Comment:
If you look at [travis docs](https://docs.travis-ci.com/user/osx-ci-environment/#OS-X-Version), they only support Xcode 8.2 beta... I would say to wait until travis fully adopt it.